apps/museums/fields/index.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
   },
   'contact-phone': {
     mixin: 'input-text',
-    validate: ['required', 'phonenumber']
+    validate: 'required'
```

Comment:
Automated testing exposed that out of the box phone number validation fails with spaces in the number. Options are:

* change the validator
* format the value to remove spaces
* remove the validator

Since other apps in firearms don't validate phone numbers, I went for the quickest option.